data/json/items/gun/40x46mm.json

```
@@ -123,6 +123,40 @@
     "pocket_data": [ { "pocket_type": "MAGAZINE", "rigid": true, "ammo_restriction": { "40x46mm": 6 } } ],
     "melee_damage": { "bash": 12 }
   },
+  {
+    "id": "rm802",
+    "copy-from": "launcher_base",
+    "looks_like": "m79",
+    "type": "GUN",
+    "name": { "str": "RCI 40x46 pump action grenade launcher" },
+    "description": "The RCI 40x46 pump action grenade launcher was the first milspec product of the Rivet & Crimp Insustries, Rivtech's predecessor.  Outdoing its competitors in tests, this launcher was adopted in 1981 to replace the aging and inadequate single shot launchers after the Vietnam War.  After it was phased out in favor of Milkor Mgl and became surplus it became the favorite of Riot Control Units and PMCs worldwide.  Its 3+1 capacity lets it maintain a good suppressing fire power and it is lightweight for its size thanks to its aluminum frame.",
```

```suggestion
    "description": "The RCI 40x46 pump action grenade launcher was the first milspec product of the Rivet & Crimp Industries, Rivtech's predecessor.  Outdoing its competitors in tests, this launcher was adopted in 1981 to replace the aging and inadequate single shot launchers after the Vietnam War.  After it was phased out in favor of Milkor MGL and became surplus it became the favorite of Riot Control Units and PMCs worldwide.  Its 3+1 capacity lets it maintain a good suppressing fire power and it is lightweight for its size thanks to its aluminum frame.",
```
When the RivTech is sus

---

damn